perl.yml

```diff
@@ -695,3 +695,230 @@ rh-perl524:
+    - perl-podlators
+    - perl-Pod-Simple
+    - perl-PadWalker
+    - perl-Module-Load
+    - perl-Term-Cap
+    - perl-Math-BigRat
+    - perl-Exporter
+    - perl-File-Temp
+    - perl-File-Path
+    - perl-Sub-Install
+    - perl-Variable-Magic
+    - perl-CPAN
+    - perl-File-Which
+    - perl-constant
+    - perl-Test-Deep
+    - perl-CPAN-Meta-Requirements
+    - perl-DB_File
+    - perl-Devel-Size
+    - perl-Locale-Maketext
+    - perl-Test-Needs
+    - perl-CPAN-Meta
+    - perl-Config-Perl-V
+    - perl-IPC-System-Simple
+    - perl-IPC-SysV
+    - perl-Devel-StackTrace
+    - perl-bignum
+    - perl-Data-Dumper
+    - perl-Digest-SHA
+    - perl-Test-Without-Module
+    - perl-File-Copy-Recursive
+    - perl-Sub-Uplevel
+    - perl-Class-Singleton
+    - perl-Time-Local
+    - perltidy
+    - perl-Number-Compare
+    - perl-inc-latest
+    - perl-parent
+    - perl-Text-Tabs+Wrap
+    - perl-DBI
+    - perl-PerlIO-via-QuotedPrint
+    - perl-Params-Util
+    - perl-Thread-Queue
+    - perl-Perl-OSType
+    - perl-Compress-Raw-Zlib
+    - perl-Text-Balanced
+    - perl-CPAN-DistnameInfo
+    - perl-Sub-Identify
+    - perl-Ref-Util
+    - mod_perl
+    - perl-Test-NoWarnings
+    - perl-Module-Build
+    - perl-App-cpanminus
+    - perl-Sub-Info
+    - perl-CPAN-Meta-Check
+    - perl-DBD-MySQL
+    - perl-DBD-SQLite
+    - perl-ExtUtils-MakeMaker-CPANfile
+    - perl-Module-Pluggable
+    - perl-Test-Warn
+    - perl-Data-OptList
+    - perl-Math-BigInt-FastCalc
+    - perl-URI
+    - perl-Package-Generator
+    - perl-Term-Table
+    - perl-Test-FailWarnings
+    - perl-DBD-Pg
+    - perl-Test-Fatal
+    - perl-SUPER
+    - perl-File-HomeDir
+    - perl-Devel-Caller
+    - perl-Text-Diff
+    - perl-File-Find-Rule
+    - perl-Exception-Class
+    - perl-File-ShareDir
+    - perl-Parse-PMFile
+    - perl-Devel-LexAlias
+    - perl-Date-ISO8601
+    - perl-Parallel-ForkManager
+    - perl-Package-Stash-XS
+    - perl-DynaLoader-Functions
+    - perl-Class-Method-Modifiers
+    - perl-Sub-Exporter
+    - perl-Module-Implementation
+    - perl-Test2-Suite
+    - perl-Test-MockModule
+    - perl-Sub-Quote
+    - perl-Unicode-UTF8
+    - perl-local-lib
+    - perl-Dist-CheckConflicts
+    - perl-Data-Section
+    - perl-Sub-Exporter-Progressive
+    - perl-Archive-Zip
+    - perl-Role-Tiny
+    - perl-Package-Stash
+    - perl-Devel-CallChecker
+    - perl-Path-Tiny
+    - perl-Eval-Closure
+    - perl-Test2-Plugin-NoWarnings
+    - perl-Specio
+    - perl-Software-License
+    - perl-Test-File-ShareDir
+    - perl-Params-Classify
+    - perl-B-Hooks-EndOfScope
+    - perl-namespace-clean
+    - perl-Params-ValidationCompiler
+    - perl-DateTime-TimeZone-SystemV
+    - perl-namespace-autoclean
+    - perl-DateTime-TimeZone-Tzfile
+    - perl-DateTime-Locale
+    - perl-DateTime-TimeZone
+    - perl-DateTime
+    - rh-perl526:
+        cmd: |
+          sed -i -e 's/^%perl_bootstrap 1$/#%%perl_bootstrap 1/' macro-build
```

Comment:
@ppisar  You do not have to set `cmd` element. Below is good enough.
Because actually 1st rh-perl526 and 2nd rh-perl526 are independent for each other by rpm-list-builder.
Those directories created to different directory.
See https://github.com/sclorg/rpm-list-builder/blob/master/docs/users_guide.md#work-directory


```
  - rh-perl526
```